server/puppeteerserver.js

```diff
@@ -0,0 +1,47 @@
+const puppeteer = require('puppeteer');
+const express = require('express');
```

Comment:
only routes should be in this file. file name should be `routes.js`

Reply:
moved functions to utill.